cluster/node-pools/worker-karpenter/provisioners.yaml

```diff
@@ -150,6 +150,14 @@ spec:
           values:
             - nitro
 #{{ end }}
+#{{ if eq .Cluster.ConfigItems.karpenter_nitro_support_required "true" }}
+#{{ if and (eq .Cluster.Provider "zalando-aws") (eq .NodePool.KarpenterInstanceTypeStrategy "not-specified") }}
```

Comment:
> when running EKS with IPv6

This line excludes EKS. Why?

Reply:
For `eks` we already have the lines right above this one, where we have enabled this already. So, my thinking is, once we move to EKS, we can simply drop this section altogether, and keep the one above it.